Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG PYTHON_VERSION=3.12.0
+FROM python:${PYTHON_VERSION}-slim as base
+LABEL maintainer="silva.vvss12@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /cinema_service
```

Comment:
There is a syntax issue with the `WORKDIR` command. It should be `WORKDIR /cinema_service` without a space between the command and the directory path.